### ISSUE TITLE
pycross_internal_repo incompatible with --incompatible_no_implicit_file_export flag

### ISSUE THREAD

**Issue by LaurenceTews** [jvolkman/rules_pycross#165]
When creating a pycross_internal_repo (in my case via rules_pycross_dependencies()), the build fails when the `--incompatible_no_implicit_file_export` Bazel flag is set. I'm not sure when this flag will be flipped on by default, but could be worth fixing. I had a dig around, and referencing the generated `:file` target in _installer_whl might be the easiest fix here (e.g. @rules_pycross_internal_deps_wheel_installer_0.7.0_py3_none_any//file:file). 

Error Message:

```
ERROR: /home/ltews/.cache/bazel/_bazel_ltews/ff6247bb8160f097b6ee98d451cc4e4d/external/rules_pycross_internal/BUILD.bazel:3:6: in alias rule @@rules_pycross_internal//:installer_whl: target '@@rules_pycross_internal_deps_wheel_installer_0.7.0_py3_none_any//file:installer-0.7.0-py3-none-any.whl' is not visible from target '@@rules_pycross_internal//:installer_whl'. Check the visibility declaration of the former target if you think the dependency is legitimate. To set the visibility of that source file target, use the exports_files() function
ERROR: /home/ltews/.cache/bazel/_bazel_ltews/ff6247bb8160f097b6ee98d451cc4e4d/external/rules_pycross_internal/BUILD.bazel:3:6: Analysis of target '@@rules_pycross_internal//:installer_whl' failed
```

The referenced alias looks like this 
```
alias(
    name = "installer_whl",
    actual = "@rules_pycross_internal_deps_wheel_installer_0.7.0_py3_none_any//file:installer-0.7.0-py3-none-any.whl",
)
```
